__tests__/index.test.js

```diff
@@ -24,3 +24,54 @@ describe("/api/topics tests", () => {
     return request(app).get("/api/unknownendpoint").expect(404);
   });
 });
+
+describe("/api tests", () => {
+  test("documents available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toHaveProperty("GET /api");
+        expect(response.body).toHaveProperty("GET /api/topics");
+      });
+  });
+});
+describe("/api/articles/:article_id tests", () => {
+  test("200: sends a single article to the client", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article).toHaveProperty("author");
+        expect(response.body.article).toHaveProperty("title");
+        expect(response.body.article).toHaveProperty("body");
+        expect(response.body.article).toHaveProperty("topic");
+        expect(response.body.article).toHaveProperty("created_at");
+        expect(response.body.article).toHaveProperty("votes");
+        expect(response.body.article).toHaveProperty("article_img_url");
```

Comment:
Using toHaveProperty isn't bad, but there is better Jest Matchers we can use for more robust and extendable testing, take a look into toMatchObjects and expect.objectContaining.